pyproject.toml

```diff
@@ -29,10 +29,11 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-  'click ~= 8.1',
-  'ghedesigner ~= 1.5',
-  'pandas ~= 2.0',
+  "click ~= 8.1",
+  "ghedesigner ~= 1.5",
+  "pandas ~= 2.0",
   "rich ~= 13.0",
+  "SecondaryCoolantProps ~= 1.3"
```

Comment:
How did you find out about an obscure, helpful package like this... oh. 😆

Reply:
It pays to work on a single topic, for like, ever.... 👴